README.md

```diff
@@ -0,0 +1,6 @@
+# org.gtk.Gtk3theme.Breeze
+### Workarounds
+- Using a custom color scheme (etc. Breeze Dark) and ascent color. Related issue [#3901](https://github.com/flatpak/flatpak/issues/3901).
+  ```sh
+  flatpak override --user --filesystem=xdg-config/gtk-3.0:ro
```

Comment:
Should this additionally include "--filesystem=xdg-config/gtk-4.0:ro"?

Reply:
That wouldn't work as flatpak doesn't allow for custom Gtk4 themes yet and KDE doesn't store any color scheme information in that folder. A new package named ``org.gtk.Gtk3theme.Breeze`` might have to be created when flatpak adds the Gtk4 functionality as well.

Reply:
@vulongm where did you get that information about Flatpak not allowing custom Gtk4 themes? I don't think anything in Flatpak itself would prevent `--filesystem=xdg-config/gtk-4.0:ro` from working, though I also don't know much about how themes work.